src/content/release-notes.mdx

```diff
@@ -5,6 +5,42 @@ sidebar_label: Release notes
 id: release-notes
 ---
 
+## 1.27.0
+
+9 December 2024
+
+This version is compatible with Kubernetes versions 1.27 to 1.30
+
+### Breaking Changes
+
+- Remove code for our deprecated Quickstarts feature <!-- 8842 -->
+
+### New Features
+
+- We now allow the ability for admins of Okteto to [set the minimum supported CLI version for their team](admin/dashboard.mdx#command-line-cli)  <!-- 8829 -->
+- We've published the Okteto Manifest JSON Schema for inline suggestions and validation for creating and editing Okteto manifests within your code editor
+- Added support for overwriting CLI images at the Helm chart level
+- Shipped additional support for the installation of [Okteto in Air-Gapped Environments](self-hosted/manage/air-gapped.mdx) <!-- 8914 -->
+- Configured buildkit probes to only accept requests when buildkit is healthy  <!-- 8871 -->
+- Added support for configuring the Okteto control plane jobs TTL at the helm level  <!-- 8917 -->
+- feat: send cli image on graphql clustermetadata  <!-- 8914 -->
```

Comment:
Not sure I understand this. Does it need a mention?

Reply:
No, I don't think this should be mentioned. It is an internal detail of implementation for the images management improvements